internal/commands/user/disable_test.go

```diff
@@ -0,0 +1,270 @@
+package user
+
+import (
+	"errors"
+	"strings"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/app"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+)
+
+func TestUserDisableSetup(t *testing.T) {
+	t.Run("should construct a realm client with the configured base url", func(t *testing.T) {
+		profile := mock.NewProfile(t)
+		profile.SetRealmBaseURL("http://localhost:8080")
+		cmd := &CommandDisable{inputs: disableInputs{}}
+
+		assert.Nil(t, cmd.realmClient)
+		assert.Nil(t, cmd.Setup(profile, nil))
+		assert.NotNil(t, cmd.realmClient)
+	})
+}
+
+func TestUserDisableHandler(t *testing.T) {
+	projectID := "projectID"
+	appID := "appID"
+	testApp := realm.App{
+		ID:          appID,
+		GroupID:     projectID,
+		ClientAppID: "eggcorn-abcde",
+		Name:        "eggcorn",
+	}
+	testUsers := []realm.User{
+		{
+			ID:         "user-1",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeAnonymous}},
+		},
+		{
+			ID:         "user-2",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeAnonymous}},
+			Disabled:   true,
+		},
+	}
+
+	for _, tc := range []struct {
+		description    string
+		disableUserErr error
+	}{
+		{description: "should disable a user when a user id is provided"},
```

Comment:
meganit, if you want to in-line these test case declarartions, omit the field names to reduce text-noise:

```go
{"should disable a user when a user id is provided", nil},
{"should save failed disable errors", errors.New("client error")},
```